apps/etterlatte-behandling/src/main/kotlin/sak/SakTilgangDao.kt

```diff
@@ -38,7 +38,7 @@ class SakTilgangDao(private val datasource: DataSource) {
     }
 
     fun hentSakMedGarderingOgSkjermingPaaBehandling(behandlingId: String): SakMedGraderingOgSkjermet? {
-        datasource.connection.use {
+        datasource.connection.use { it ->
```

Comment:
Bruk heller `connection` eller noe som på linje 27. Eventuelt bare dropp det og bruk `it` som det er ment å brukes.

Reply:
Det skulle vært conncetion, må ha vært skriveleif. Fikser